app/lib/clients/vault/entity.rb

```diff
@@ -19,11 +62,16 @@ def delete_entity(name)
       def get_entity_data(sub)
         entity = read_entity(sub)
         if entity.nil?
-          [ [], nil ]
+          [ [], {} ]
         else
           [ entity.data[:policies], entity.data[:metadata] ]
         end
       end
+
+      def read_group_alias(entity_name, alias_name)
+        id = read_entity_alias_id(entity_name, alias_name)
+        client.logical.read("identity/entity-alias/id/#{id}")
+      end
```

Comment:
I think all uses of "entity" above should be "group"

Reply:
fixed